tests/test_dsl.py

```diff
@@ -320,3 +326,273 @@ def assert_cohorts_equivalent(dsl_cohort, qm_cohort):
     # ...and if the columns are the same.
     for k in dsl_col_defs:
         assert repr(dsl_col_defs[k]) == repr(qm_col_defs[k])
+
+
+def test_boolseries_and(bool_series):
+    series1 = bool_series()
+    series2 = bool_series()
+
+    expected = Comparator(
+        connector="and_",
+        negated=False,
+        operator=None,
```

Comment:
You can drop the `negated` and `operator` arguments, since they're the default.

Reply:
Done and done, got all the `connector=None`s too.